### ISSUE TITLE
Not visible to VoiceOver

### ISSUE THREAD

**Issue by michaeljtsai** [shpakovski/MASShortcut#152]
I think this used to work, but as of now (compiling with Xcode 12.3, running on macOS 10.15.7 and 11.2 beta), it’s not possible to select the `MASShortcutView` using VoiceOver, and I don’t hear the accessibility notifications when the shortcut is changed. I think I was able to fix this by overriding `-[MASShortcutView isAccessibilityElement]` to return `YES`. The [documentation](https://developer.apple.com/documentation/appkit/nsaccessibility/1535002-accessibilityelement) suggests that this is necessary, so I don’t understand why it was working for me before without doing this.

**Comment by shpakovski** [comment]
@michaeljtsai Thanks a lot for the heads up! Could you please make a pull request? 🙏

**Comment by michaeljtsai** [comment]
I was hoping someone who knows more about this would have insight into why it worked before without this and therefore whether this is in fact the right fix. But, yes, I can make a PR:

https://github.com/shpakovski/MASShortcut/pull/153

**Comment by Kentzo** [comment]
@michaeljtsai Probably Apple used to have additional checks that were not documented (e.g. by explicitly looking for other accessibility methods in the interface).

**Comment by michaeljtsai** [comment]
That works, too.
